### PR TITLE
DOC: Add inclusively focused language and links to the template

### LINKED ISSUE #349: Add inclusively focused language and links to the template
I.e. the imposter syndrome disclaimer that yt have in their readme and CoC etc etc

### PR DESCRIPTION
**Pull request by pllim**
Fix #349

Once, we hash things out here, we should then discuss about including it (or not) in Astropy core and other existing affiliated packages.

cc @Cadair , @mwcraig , @bsipocz , and @astropy/coordinators 

Background: This stemmed from diversity discussions at Scipy 2018 and was given permission by Nathan Goldbaum from `yt` to share.

### CODE REVIEW

**Comment by mwcraig** [comment]
This looks really good to me!

File: {{ cookiecutter.package_name }}/README.rst [hunk 1]
```diff
@@ -16,3 +16,36 @@ the terms of the {{ cookiecutter.license }} license. This package is based upon
...
+project (and yes - that includes giving feedback about the contribution
+process). Some of these contributions may be the most valuable to the project as
+a whole, because you're coming to the project with fresh eyes, so you can see
+the errors and assumptions that seasoned contributors have glossed over.
+
+(This disclaimer was originally written by
+`Adrienne Lowe <https://github.com/adriennefriend>`_ for a
+`PyCon talk <https://www.youtube.com/watch?v=6Uj746j9Heo>`_, and was adapted by
+{{ cookiecutter.package_name }} based on its use in the README file for the
+`MetPy project <https://github.com/Unidata/MetPy>`_.)
```
Comment (astrofrog) [inline {{ cookiecutter.package_name }}/README.rst]: MInor: What about making this italics instead of being in brackets?

Comment (pllim) [reply {{ cookiecutter.package_name }}/README.rst]: Sure!

**Comment by astrofrog** [comment]
Just a thought - should this be in the ``CONTRIBUTING.md`` file rather than the README? Or is the issue that it won't be as visible?

**Comment by bsipocz** [comment]
@astrofrog - What about having it in both (or cross linking)? I feel `CONTRIBUTING` is the right place pedantically, but then putting it there already work with the assumption that newcomers know how things work.

**Comment by astrofrog** [comment]
I'd rather not duplicate text - maybe we can just leave it in the README and then package maintainers can choose whether to add a CONTRIBUTING file?

**Comment by kelle** [comment]
I don’t have a strong opinion on this particular issue BUT in general, I think it’s good to repeat info. It’s hard to figure out how things work and the least we can do is have useful info in a couple places.

**Comment by kelle** [comment]
Put another way, not having duplicated text feels cleaner to us but having it makes it easier for the user/contributor. So, in this case, I vote to err on the side of user. At least until our contributor docs and workflow become super easy and intuitive to find and comprehend. :)

**Comment by pllim** [comment]
I see two approvals but no merge. Is there something else I need to do here? I see some discussions above but the conclusion is unclear to me. Please advise. Thanks!

**Comment by astrofrog** [comment]
I'll just merge this for now - the README is the most visible place to put this. If anyone feels strongly about the CONTRIBUTING file, feel free to open a new PR :)

**Pull request merged**